adapter/internal/messaging/notification_listener.go

```diff
@@ -442,12 +444,28 @@ func handlePolicyEvents(data []byte, eventType string) {
 			return
 		}
 
-		subscriptionPolicy := types.SubscriptionPolicy{ID: subscriptionPolicyEvent.PolicyID, TenantID: -1,
-			Name: subscriptionPolicyEvent.PolicyName, QuotaType: subscriptionPolicyEvent.QuotaType,
+		subscriptionPolicy := types.SubscriptionPolicy{
+			ID:                   subscriptionPolicyEvent.PolicyID,
+			TenantID:             TenantID,
```

Comment:
We can remove TenantID from CC. Are we processing it?
@malinthaprasan wdyt?

Reply:
TenantId will always be -1234 (super tenant) in Choreo. In some cases I have seen it's 0 in some variables when it's not set properly. Anyway there's no use case to process the tenant ID in Choreo. If we can remove it without breaking the code, +1 to do it.

Reply:
Check how tenant ID parsed in other event types ?

Reply:
Checked during the PR review.